benches/trace.rs

```diff
@@ -64,13 +67,13 @@ fn criterion_benchmark(c: &mut Criterion) {
     });
 }
 
-fn trace_benchmark_group<F: Fn(&sdk::Tracer)>(c: &mut Criterion, name: &str, f: F) {
+fn trace_benchmark_group<F: Fn(&sdk::trace::Tracer)>(c: &mut Criterion, name: &str, f: F) {
```

Comment:
These are kinda unergonomic now with the double module prefix, but not sure how to avoid that as they share names with the traits. Any ideas here? or is this the best we can do currently.

Reply:
Looks like go uses the equivalent of:

```rust
use opentelemetry::sdk::trace as sdktrace;
// ...
let tracer: sdktrace::Tracer;

// or
 
use opentelemetry::sdk::trace::Tracer as SdkTracer;
// ...
let tracer: SdkTracer;
```

Alternatively we could have a convention like our examples always import `api::Tracer as _` and `Tracer` is always in reference to sdk, or vice versa.

Reply:
I mean, we _could_ `pub use` it in `src/sdk/mod.rs`, but then we go back to the inconsistency. It's weird, but if we flatten this into `sdk`, then do we do the same for any type that is prefixed with `Trace*`? And then we're back where we started. 🤷

Reply:
Yeah, I think example aliases are probably good for now.

Reply:
@jtescher updated examples, doc tests and tracing bench to use the alias.

Reply:
Ok cool, the aliases are not breaking changes so looks good to me even if we want to experiment here